### PR TITLE
[DOCFIX] Improve docs for HDFS and Mapreduce

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

**Comment by ifcharming** [comment]
@apc999 @calvinjia

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ of Alluxio source code.
 * Modify the `hadoop.version` tag defined in `${ALLUXIO_HOME}/pom.xml`. E.g., to work with Hadoop
 `2.6.0`, modify this pom file to set "`<hadoop.version>2.6.0</hadoop.version>`" instead of
 "`<hadoop.version>2.2.0</hadoop.version>`". Then recompile the source using maven.
+To make compiling faster, you can add `-DskipTests` option to skip unit tests.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: Let's add the skip find bugs / checkstyle as well?

Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: Hmm, test should be the bottleneck to build Alluxio. I think find bugs/ checkstyle should be pretty fast.
In other docs, only `-DskipTests` is applied. Do we want keep the sample command elegant, with as few options as possible?

**Comment by apc999** [comment]
@ifcharming  can you reopen this PR to branch-1.1 instead of master? thanks

**Pull request closed without merging**